client/my-sites/hosting/hosting-activate-status.tsx

```diff
@@ -44,9 +45,6 @@ const HostingActivateStatus = ( {
 		if ( ! isTransferring && wasTransferring && isTransferCompleted ) {
 			setWasTransferring( false );
 		}
-		if ( ! isTransferCompleted ) {
-			dispatch( fetchAutomatedTransferStatus( siteId ?? 0 ) );
```

Comment:
The redux state is updated using this method, we basically updating redux with each status update. If we don't do this then the redux state is not updated which cause the `transferState` to be in wrong state.